src/Azure.Functions.Cli/Actions/AzureActions/CreateAADApplication.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Threading.Tasks;
+using Azure.Functions.Cli.Common;
+using Azure.Functions.Cli.Interfaces;
+using Fclp;
+namespace Azure.Functions.Cli.Actions.AzureActions
+{
+    // Invoke via `func azure auth create-aad -aad-name {displayNameOfAAD} --app-name {displayNameOfApp}`
+    [Action(Name = "create-aad", Context = Context.Azure, SubContext = Context.Auth, HelpText = "Creates a production Azure Active Directory application with given name. Links it to specified Azure Application")]
+    class CreateAADApplication : BaseAzureAction
```

Comment:
so the difference between `CreateAADApplication.CreateAADApplication` and `AuthActions.CreateAADApplication` is that the latter assumes `localhost` while the former sets it up for your `app.azurewebsites.net`? If so, you probably could combine them into one Action and have `AppName` default to `localhost` if not specified, right?

Reply:
Yes, you're correct. It's currently two separate commands based on a meeting I had with @mattchenderson, but combining them would be trivial. I'm happy to change it if it seems like that would provide a better UX :)